vespa-http-client/src/main/java/com/yahoo/vespa/http/client/runner/CommandLineArguments.java

```diff
@@ -192,6 +198,11 @@ int getWhenVerboseEnabledPrintMessageForEveryXDocuments() {
 
     SessionParams createSessionParams(boolean useJson) {
         final int minThrottleValue = useDynamicThrottlingArg ? 10 : 0;
+        ConnectionParams.Builder connectionParamsBuilder = new ConnectionParams.Builder();
+        for (String header : headers) {
+            String[] nameAndValue = header.split(":");
+            connectionParamsBuilder.addHeader(nameAndValue[0].trim(), nameAndValue[1].trim());
```

Comment:
Consider triggering a better error message if a malformed header argument is passed. Right now it's going to be an `ArrayIndexOutOfBoundsException`